SimPEG/PF/Gravity.py

```diff
@@ -447,3 +456,94 @@ def readUBCgravObs(obs_file):
     survey.dobs = d
     survey.std = wd
     return survey
+
+
+class Problem3D_Diff(Problem.BaseProblem):
```

Comment:
same question here. Do we want these changes included in the dcip pr? cc @fourndo